themes/default/content/docs/intro/concepts/resources/options/additionalsecretoutputs.md

```diff
@@ -81,3 +81,7 @@ resources:
 {{< /chooser >}}
 
 Only top-level resource properties can be designated secret. If sensitive data is nested inside of a property, you must mark the entire top-level output property as secret.
+
+{{% notes "warning" %}}
```

Comment:
If you want to use warning level, these'll need to be tweaked, as right now they're appearing as regular notes:

 ```suggestion
{{% notes type="warning" %}}
```